notebooks/00-jupyter_introduction.md

```diff
@@ -216,7 +224,7 @@ Create a new cell below with the key B... when in *command* mode
 
 +++
 
-## CTRL + SHIFT + P
+## CTRL + SHIFT + C
```

Comment:
I suppose this is meant to be the command palette, which is ctrl-shift-C for me (ctrl-shirt-P opens a private window in firefox)